app/controllers/application_controller.rb

```diff
@@ -68,121 +73,184 @@ def render_no_content(**kwargs)
   def render_forbidden(**kwargs)
     skip_verify_authorized!
 
-    render status: :forbidden, json: {
-      meta: { id: request.request_id },
-      errors: [{
-        title: "Access denied",
-        detail: "You do not have permission to complete the request",
-        **kwargs,
-      }],
-    }
+    respond_to do |format|
+      format.any(:jsonapi, :json, :binary) {
```

Comment:
Is there a better way to do this? Essentially, I want `format.any(except: :html)`.